CONTRIBUTING.md

```
@@ -0,0 +1,281 @@
+kind version
+kubectl version --client
+helm version
+yamllint --version
+kubeconform -v
+```
+
+---
+
+## Repository Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)